nvi-evaluator/src/main/java/no/sikt/nva/nvi/evaluator/calculator/PointCalculator.java

```diff
@@ -90,31 +90,37 @@ public static PointCalculation calculatePoints(JsonNode jsonNode,
         massiveHackToFixObjectsWithMultipleTypes(jsonNode);
         var publicationChannel = extractChannel(instanceType, jsonNode);
         return calculatePoints(nviCreatorsWithInstitutionIds, instanceType, publicationChannel,
-                               getInstanceTypeAndLevelPoints(instanceType, publicationChannel),
                                isInternationalCollaboration(jsonNode), countCreatorShares(jsonNode));
     }
 
     private static PointCalculation calculatePoints(Map<URI, List<URI>> nviCreatorsWithInstitutionIds,
                                                     InstanceType instanceType, Channel channel,
-                                                    BigDecimal instanceTypeAndLevelPoints,
                                                     boolean internationalCollaboration, int creatorShareCount) {
+        var basePoints = getInstanceTypeAndLevelPoints(instanceType, channel);
+        var collaborationFactor = getInternationalCollaborationFactor(internationalCollaboration);
+        var institutionPoints = calculatePointsForAllInstitutions(basePoints, creatorShareCount,
+                                                                  internationalCollaboration,
+                                                                  nviCreatorsWithInstitutionIds);
+        var totalPoints = calculateTotalPoints(collaborationFactor, basePoints, BigDecimal.valueOf(creatorShareCount));
```

Comment:
Not entirely sure why creatorShareCount is `int` and not `BigDecimal` in the method signature